### PR TITLE
Feature/multiple file upload

### PR DESCRIPTION
**Pull request by Mohakjuneja**
I've started extending the functionality to allow users to upload multiple images. While I've only written a couple of test cases so far, I'm unable to call super for the "mount_uploaders" method leading to the failure of the second test case. In order to continue, I need someone to review the work and guide me in the right direction. Thanks!

### CODE REVIEW

File: lib/carrierwave/mongoid.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@
 require 'mongoid-grid_fs'
 require 'carrierwave'
 require 'carrierwave/validations/active_model'
+require 'byebug'
```
Comment (rmm5t) [inline lib/carrierwave/mongoid.rb]: You forgot this.

Comment (Mohakjuneja) [inline lib/carrierwave/mongoid.rb]: I use and 'require' byebug locally to debug. I didn't want to pollute the code for you with it so I removed it before pushing the code.

File: lib/carrierwave/mongoid.rb [hunk 2]
```diff
@@ -104,9 +105,126 @@ def serializable_hash(options=nil)
         end
       RUBY
     end
+
+    def mount_uploaders(column, uploader=nil, options={}, &block)
+      field options[:mount_on] || column, type: Array, :default => []
+
+      # super
```
Comment (Mohakjuneja) [inline lib/carrierwave/mongoid.rb]: The problem seems to be here. If I uncomment it, it returns this error-
` super: no superclass method `mount_uploaders' for MongoUser:Class`

File: lib/carrierwave/mongoid.rb (hunk 1 above)
Comment (rmm5t) [inline lib/carrierwave/mongoid.rb]: But this shows that you **did not** fully remove it before pushing. It's why the entire build is failing on Travis at the moment.

Comment (Mohakjuneja) [inline lib/carrierwave/mongoid.rb]: I apologize. I must have missed it. I've updated the file again

**Comment by Mohakjuneja** [comment]
I want to see this feature through but I'm going to need some assistance. Can anyone get me past the error I've mentioned above?

File: lib/carrierwave/mongoid.rb (hunk 2 above)
Comment (zaithottakath) [inline lib/carrierwave/mongoid.rb]: The current version of `carrierwave-mongoid` depends on `"carrierwave", [">= 0.8.0", "< 0.12.0"]` https://github.com/carrierwaveuploader/carrierwave-mongoid/blob/v0.10.0/carrierwave-mongoid.gemspec#L23. The greatest version of `carrierwave` that falls within those bounds is `0.11.2` which does not have a `mount_uploaders` method defined within `CarrierWave::Mount` https://github.com/carrierwaveuploader/carrierwave/blob/v0.11.2/lib/carrierwave/mount.rb#L137. So there is no `mount_uploaders` method to call `super` on. It seems that this gem needs to be updated to work with `carrierwave 1.0.0` #172 which has the `mount_uploaders` method https://github.com/carrierwaveuploader/carrierwave/blob/v1.0.0/lib/carrierwave/mount.rb#L287

**Comment by mshibuya** [comment]
Closing in favor of #199.

**Pull request closed without merging**